bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx126beta-de.html

```diff
@@ -0,0 +1,51 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}What’s new with Firefox{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_126beta') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-sm">
+      <aside class="wnp-swap">
+        <div class="wnp-swap-blurb-here">
+          <span class="wnp-swap-text-here" lang="es-ES">hola 🥘</span>
+          {% include 'firefox/whatsnew/includes/fx126beta/arrow-blue.svg' %}
+        </div>
+        <div class="wnp-swap-blurb-there">
+          <span class="wnp-swap-text-there" lang="de">hallo 🥨</span>
+          {% include 'firefox/whatsnew/includes/fx126beta/arrow-white.svg' %}
+        </div>
+      </aside>
+
+      <h2>Sag <span lang="es-ES">Hola</span> zu Firefox&nbsp;translations</h2>
```

Comment:
**Question: ** I'm noticing the `&nbsp;` unicode on each file. Is this intentional?

Reply:
**Nit:** On smaller screens, for all locales, the string starts to get off-screen a bit

![Screenshot 2024-04-11 at 4 03 41 PM](https://github.com/mozilla/bedrock/assets/42309026/c9b5983c-f529-498e-83f3-d3a2c33b433c)

Reply:
I think the `&nbsp;` was to prevent the text wrapping and keep "Firefox translations" on one line. It does make problems on mobile but this page isn't really seen on mobile so maybe we can get away with it.